<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>672</width>
    <height>553</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="maximumSize">
       <size>
        <width>1000</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QTabWidget" name="tabWidget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab">
          <attribute name="title">
           <string>Buscar por Especialidad</string>
          </attribute>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QFrame" name="frame_2">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="QFrame" name="frame_3">
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>60</height>
                 </size>
                </property>
                <property name="frameShape">
                 <enum>QFrame::StyledPanel</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Raised</enum>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout_4">
                 <item>
                  <spacer name="horizontalSpacer">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>69</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item>
                  <widget class="QComboBox" name="comboEspecialidad">
                   <property name="minimumSize">
                    <size>
                     <width>180</width>
                     <height>0</height>
                    </size>
                   </property>
                   <item>
                    <property name="text">
                     <string>-- Seleccione una especialidad --</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>Sistemas</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>Redes</string>
                    </property>
                   </item>
                  </widget>
                 </item>
                 <item>
                  <spacer name="horizontalSpacer_2">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>69</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item>
                  <widget class="QPushButton" name="btnBuscarEspecialidad">
                   <property name="minimumSize">
                    <size>
                     <width>150</width>
                     <height>0</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>Buscar</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <spacer name="horizontalSpacer_3">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>69</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QFrame" name="frame_4">
                <property name="frameShape">
                 <enum>QFrame::StyledPanel</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Raised</enum>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout_5">
                 <item>
                  <widget class="QTableWidget" name="tbl_alumno">
                   <attribute name="horizontalHeaderDefaultSectionSize">
                    <number>142</number>
                   </attribute>
                   <column>
                    <property name="text">
                     <string>Nombre</string>
                    </property>
                   </column>
                   <column>
                    <property name="text">
                     <string>Apellido</string>
                    </property>
                   </column>
                   <column>
                    <property name="text">
                     <string>Genero</string>
                    </property>
                   </column>
                   <column>
                    <property name="text">
                     <string>Especialidad</string>
                    </property>
                   </column>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_2">
          <attribute name="title">
           <string>Buscar por Alumno</string>
          </attribute>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QFrame" name="frame_5">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="QFrame" name="frame_6">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>150</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>150</height>
                 </size>
                </property>
                <property name="frameShape">
                 <enum>QFrame::StyledPanel</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Raised</enum>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout_9">
                 <item>
                  <widget class="QFrame" name="frame_8">
                   <property name="maximumSize">
                    <size>
                     <width>211</width>
                     <height>209</height>
                    </size>
                   </property>
                   <property name="frameShape">
                    <enum>QFrame::StyledPanel</enum>
                   </property>
                   <property name="frameShadow">
                    <enum>QFrame::Raised</enum>
                   </property>
                   <layout class="QVBoxLayout" name="verticalLayout_3">
                    <item>
                     <widget class="QLineEdit" name="txtCodigo">
                      <property name="maximumSize">
                       <size>
                        <width>198</width>
                        <height>16777215</height>
                       </size>
                      </property>
                      <property name="font">
                       <font>
                        <pointsize>11</pointsize>
                       </font>
                      </property>
                      <property name="placeholderText">
                       <string>ID Codigo</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QPushButton" name="btnBuscarAlumno">
                      <property name="minimumSize">
                       <size>
                        <width>150</width>
                        <height>0</height>
                       </size>
                      </property>
                      <property name="text">
                       <string>Buscar Alumno</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <spacer name="horizontalSpacer_4">
                      <property name="orientation">
                       <enum>Qt::Vertical</enum>
                      </property>
                      <property name="sizeHint" stdset="0">
                       <size>
                        <width>20</width>
                        <height>91</height>
                       </size>
                      </property>
                     </spacer>
                    </item>
                   </layout>
                  </widget>
                 </item>
                 <item>
                  <widget class="QFrame" name="frame_9">
                   <property name="maximumSize">
                    <size>
                     <width>500</width>
                     <height>16777215</height>
                    </size>
                   </property>
                   <property name="frameShape">
                    <enum>QFrame::StyledPanel</enum>
                   </property>
                   <property name="frameShadow">
                    <enum>QFrame::Raised</enum>
                   </property>
                   <widget class="QLabel" name="label_10">
                    <property name="geometry">
                     <rect>
                      <x>20</x>
                      <y>10</y>
                      <width>61</width>
                      <height>20</height>
                     </rect>
                    </property>
                    <property name="text">
                     <string>Nombre:</string>
                    </property>
                   </widget>
                   <widget class="QLabel" name="txtNombre">
                    <property name="geometry">
                     <rect>
                      <x>90</x>
                      <y>10</y>
                      <width>431</width>
                      <height>20</height>
                     </rect>
                    </property>
                    <property name="font">
                     <font>
                      <pointsize>10</pointsize>
                     </font>
                    </property>
                    <property name="styleSheet">
                     <string notr="true">background-color: rgb(178, 255, 198);</string>
                    </property>
                    <property name="text">
                     <string> ----</string>
                    </property>
                   </widget>
                   <widget class="QLabel" name="label_9">
                    <property name="geometry">
                     <rect>
                      <x>20</x>
                      <y>80</y>
                      <width>71</width>
                      <height>20</height>
                     </rect>
                    </property>
                    <property name="text">
                     <string>Especialidad:</string>
                    </property>
                   </widget>
                   <widget class="QLabel" name="txtEspecialidad">
                    <property name="geometry">
                     <rect>
                      <x>90</x>
                      <y>80</y>
                      <width>191</width>
                      <height>20</height>
                     </rect>
                    </property>
                    <property name="font">
                     <font>
                      <pointsize>10</pointsize>
                     </font>
                    </property>
                    <property name="styleSheet">
                     <string notr="true">background-color: rgb(178, 255, 198);</string>
                    </property>
                    <property name="text">
                     <string>  ----</string>
                    </property>
                   </widget>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="Line" name="line_4">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QFrame" name="frame_7">
                <property name="frameShape">
                 <enum>QFrame::StyledPanel</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Raised</enum>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout_8">
                 <item>
                  <widget class="QTableWidget" name="tblNotas">
                   <property name="minimumSize">
                    <size>
                     <width>570</width>
                     <height>0</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>570</width>
                     <height>16777215</height>
                    </size>
                   </property>
                   <attribute name="horizontalHeaderDefaultSectionSize">
                    <number>110</number>
                   </attribute>
                   <column>
                    <property name="text">
                     <string>Curso</string>
                    </property>
                   </column>
                   <column>
                    <property name="text">
                     <string>Nota 1</string>
                    </property>
                   </column>
                   <column>
                    <property name="text">
                     <string>Nota 2</string>
                    </property>
                   </column>
                   <column>
                    <property name="text">
                     <string>Promedio</string>
                    </property>
                   </column>
                   <column>
                    <property name="text">
                     <string>Condicion</string>
                    </property>
                   </column>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>672</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
